scripts/Testing/Testcases/config_tests.xml

```diff
@@ -276,6 +276,7 @@
   <test NAME="SEQ">
     <DESC>sequencing bfb test (10 day seq,conc tests)</DESC>
     <INFO_DBUG>1</INFO_DBUG>
+    <BFBFLAG>TRUE</BFBFLAG>
```

Comment:
The coupler - if the number of coupler tasks change between tests this needs to be set to true to get bfb results.